app/lib/features/spaces/widgets/public_spaces_selector.dart

```diff
@@ -206,71 +207,80 @@ class PublicSpaceItem extends ConsumerWidget {
+              child: ListTile(
+                onTap: () => onSelected(
+                  space,
+                  withInfo.valueOrNull,
+                ),
+                leading: withInfo.when(
+                  data: (data) => data != null
+                      ? ActerAvatar(
+                          mode: DisplayMode.Space,
+                          uniqueId: spaceId,
+                          displayName: data.spaceProfileData.displayName,
+                          avatar: data.spaceProfileData.hasAvatar()
+                              ? data.spaceProfileData.getAvatarImage()
+                              : null,
+                        )
+                      : fallbackAvatar(),
+                  error: (e, a) {
+                    debugPrint('loading failed: $e');
+                    return ActerAvatar(
+                      mode: DisplayMode.Space,
+                      uniqueId: spaceId,
+                      displayName: spaceId,
+                    );
+                  },
+                  loading: fallbackAvatar,
+                ),
+                title: Text(
+                  space.name() ?? 'no display name',
+                  style: Theme.of(context).textTheme.labelLarge,
+                  softWrap: false,
+                ),
+                subtitle: Text(
+                  '${space.numJoinedMembers()} Members',
+                  style: Theme.of(context).textTheme.labelSmall,
+                  softWrap: false,
+                ),
+                trailing: withInfo.when(
+                  data: (data) => data != null
+                      ? const Chip(label: Text('member'))
+                      : space.joinRuleStr() == 'Public'
+                          ? OutlinedButton(
+                              onPressed: () => onSelected(
+                                space,
+                                withInfo.valueOrNull,
+                              ),
+                              child: const Text('join'),
+                            )
+                          : OutlinedButton(
+                              onPressed: () => onSelected(
+                                space,
+                                withInfo.valueOrNull,
+                              ),
+                              child: const Text('request to join'),
                             ),
-                            child: const Text('request to join'),
-                          ),
-                error: (e, s) => Text('error loading membership: $e'),
-                loading: () => const Text('loading'),
+                  error: (e, s) => Text('error loading membership: $e'),
+                  loading: () => const Text('loading'),
+                ),
               ),
             ),
           ),
-          Padding(
-            padding: const EdgeInsets.only(
-              left: 20,
-              top: 2,
-              bottom: 2,
+          Flexible(
+            child: Html(
```

Comment:
do we allow html in the topic? That sounds dangerous... especially to use here ... 5 lines ... that is a lot of text for a rather small widget...

yeah. no... I looked it up, we aren't allowing `space.topic()` to use HTML in the about-card either. Please revert this.